### PR TITLE
Support for transcribe endpoint

### PR DESCRIPTION
**Pull request by nliu96**

### CODE REVIEW

File: cohere/client.py [hunk 1]
```diff
@@ -220,6 +221,10 @@ def detokenize(self, tokens: List[int]) -> Detokenization:
         })
         return Detokenization(_future=self._executor.submit(self.__request, json_body, cohere.DETOKENIZE_URL))
 
+    def whisper(self, file_paths: List[str]) -> Whispers:
+        response = self.__file_request(file_paths, cohere.WHISPER_URL)
```
Comment (CoderHam) [inline cohere/client.py:225]: Handle error when file_paths is not a list of strings

File: cohere/__init__.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 EMBED_URL = 'embed'
 CLASSIFY_URL = 'classify'
 EXTRACT_URL = 'extract'
+WHISPER_URL = 'whisper'
```
Comment (CoderHam) [inline cohere/__init__.py:11]: In blobheart the endpoint is currently 'speech'
We should align to one of these

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: cohere/client.py (hunk 1 above)
Comment (nliu96) [reply cohere/client.py]: What is the best way to check typing. You can't check if it's a list of strings with isInstance easily.

File: cohere/__init__.py (hunk 2 above)
Comment (nliu96) [reply cohere/__init__.py]: Renamed to speech.

File: cohere/client.py (hunk 1 above)
Comment (CoderHam) [reply cohere/client.py]: Yes
```
if isinstance(value, List):
```

File: cohere/__init__.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 EMBED_URL = 'embed'
 CLASSIFY_URL = 'classify'
 EXTRACT_URL = 'extract'
+SPEECH_URL = 'whisper/speech'
```
Comment (CoderHam) [inline cohere/__init__.py:11]: Should this not be just "speech'?

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: cohere/__init__.py (hunk 3 above)
Comment (CoderHam) [reply cohere/__init__.py:11]: The model name should move to the co.speech command to be symmetric with the rest of the endpoints.

Comment (nickfrosst) [reply cohere/__init__.py:11]: i think co.transcribe would be more symmetric with the other endpoints, as all our other endpoints are verbs.

Comment (CoderHam) [reply cohere/__init__.py]: Made that fix 🎆

File: cohere/__init__.py [hunk 4]
```diff
@@ -8,6 +8,7 @@
 EMBED_URL = 'embed'
 CLASSIFY_URL = 'classify'
 EXTRACT_URL = 'extract'
+TRANSCRIBE_URL = 'whisper/transcribe'
```
Comment (jpekmez) [inline cohere/__init__.py:11]: you can use 'transcribe'
```suggestion
TRANSCRIBE_URL = 'transcribe'
```

Comment (CoderHam) [reply cohere/__init__.py]: Done :)

**Review by CoderHam — APPROVED** [review]

**Comment by nliu96** [comment]
Closing since not needed right now

**Pull request closed without merging**